websockets/referrer.any.js

```diff
@@ -0,0 +1,9 @@
+// META: script=constants.js?pipe=sub
+
+async_test(t => {
+  const ws = new WebSocket(SCHEME_DOMAIN_PORT + "/referrer");
+  ws.onmessage = t.step_func_done(e => {
+    assert_equals(e.data, "MISSING AS PER FETCH");
+    ws.close();
+  });
```

Comment:
You can do this to avoid timeouts
```
  ws.onclose = t.unreached_func('close');
  ws.onerror = t.unreached_func('error');
```
Locally I get an error event with chrome canary running referrer.any.html. I haven't debugged why that is. Can you reproduce though?

Reply:
I cannot, Chrome 64.0.3271.0 on macOS.

Reply:
Ah I see what I did wrong. I didn't restart wptserve after checking out your branch, so the websocket handler `/referrer` didn't exist. Sorry. Now it passes for me locally.

Reply:
Ah yeah, took a while to figure that one out too.